static/ansi-to-html.ts

```diff
@@ -476,53 +437,71 @@ function tokenize(text, options, callback) {
     return results1;
 }
 
+/**
+ * A sticky stack element
+ */
+type stickyStackElement = {
+    token: string,
+    data: number | string,
+    category: string
+};
+
+/**
+ * The type of the sticky stack
+ */
+type stickyStack_t = stickyStackElement[];
```

Comment:
CamelCase - Can probably drop stickyStack_t and use StickyStackElement[] everywhere